File: utilities/src/main/kotlin/lunalauf/rms/utilities/network/communication/message/request/RoundCountRequest.java
package lunalauf.rms.utilities.network.communication.message.request;

import com.google.gson.annotations.SerializedName;
import lunalauf.rms.utilities.network.communication.message.type.RequestType;

public class RoundCountRequest extends Request {

    @SerializedName("runner-id")
    public long runnerId;

    public RoundCountRequest() {
        super(RequestType.ROUND_COUNT);
    }

}

File: utilities/src/main/kotlin/lunalauf/rms/utilities/network/communication/message/request/RoundCountRequest.kt
package lunalauf.rms.utilities.network.communication.message.request

import com.google.gson.annotations.SerializedName
import lunalauf.rms.utilities.network.communication.message.type.RequestType

class RoundCountRequest : Request(RequestType.ROUND_COUNT) {
    @SerializedName("runner-id")
    var runnerId: Long = 0
}
